src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -515,12 +516,12 @@ public void setTimeZone(ZoneId timeZone) {
         this.timeZone = timeZone;
     }
 
-    public int getGracePeriod() {
-        return gracePeriod;
+    public int getGracePeriodMinutes() {
+        return (int) gracePeriod.toMinutes();
```

Comment:
Should this be changed to `long` (to match the return type of `Duration.toMinutes()`) or is it too much hassle? It's not a big deal cuz I don't think we'll ever encounter a situation where we'll overflow `int`...

Reply:
At first I was trying to avoid changing the field type in the entity. But I do see now that the underlying data type in the datastore is `int64` ([link](https://cloud.google.com/datastore/docs/concepts/entities#integer)), so there should be no problem with that.